README.md

```diff
@@ -50,7 +50,7 @@ portForwarding["Port Forwarding"]
 Its main responsibility is to respond to the handshake request from the `host-switch.exe`, create a network namespace and start the `vm-switch` subprocess in the newly created network namespace. In addition, it also calls unshare with provided arguments through `--unshare-args`. Below is a sequence diagram demonstrating the process.
 
 ## vm-switch
-Once the `vm-switch` process starts in the new namespace, it creates a tap (`eth0`) and a `lo` device. The Kernel then forwards all the raw Ethernet frames to the tap device. The tap device forwards the frames over [vsock](https://wiki.qemu.org/Features/VirtioVsock) to the host. The process on the host (`host-switch.exe`) decapsulates the frames, and since it maintains both internal (`vm-switch` to `host-switch.exe`) and external (`host-switch.exe` to the internet) connections, it connects to the external endpoints via normal syscalls.
+Once the `vm-switch` process starts in the new namespace, it creates a tap (`eth0`) and a `lo` device. In addition, it creates a series of iptables rules to allow the traffic that is destined for `lo` interface to get picked up by the tap device `eth0`. The Kernel then forwards all the raw Ethernet frames to the tap device. The tap device forwards the frames over [vsock](https://wiki.qemu.org/Features/VirtioVsock) to the host. The process on the host (`host-switch.exe`) decapsulates the frames, and since it maintains both internal (`vm-switch` to `host-switch.exe`) and external (`host-switch.exe` to the internet) connections, it connects to the external endpoints via normal syscalls.
```

Comment:
https://github.com/rancher-sandbox/rancher-desktop-networking/blob/76945a86232f720e9936682ee0fb9047d85b6c3c/cmd/vm/switch_linux.go#L158-L162

The rules sound like they're doing the _opposite_ of what's described here? That is, I parse the README changes to mean:
1. Within the network namespace, I run `curl http://localhost:12345`
2. The iptables rules redirects things, as if I ran `curl http://<eth0>:12345`.

Reply:
Yes, I will reword it.